### ISSUE TITLE
Update README.MD to reflect the credit card requirement.

### ISSUE THREAD

**Issue by mrmakeit** [mdn/discord#150]
With worker.js, the minimum cost to test the app is now $32, since worker needs it's own dynamo.  We should point this out in the README/testing documentation, since this will require adding a credit card.

**Comment by openjck** [comment]
Nice catch. At the moment, *README.md* mainly focuses on using Discord and *TESTING.md* mainly focuses on running unit and code quality tests. Would *CONTRIBUTING.md* be a good place for this? We have a note [here](https://github.com/mdn/discord/commit/9c98eb2115d61f0222a12a5e36cbab54bc3fa4d4#diff-6a3371457528722a734f3c51d9238c13R14) but could always expand that explanation.

**Comment by mrmakeit** [comment]
Right, but the redis doesn't cause a charge, the dynamo will result in a
charge, and would require making sure the dynamos are shutdown after
testing, less they charge the full $32.
CONTRIBUTING.md makes sense, since that is where the link for PR and Issues
goes.

On Fri, Aug 14, 2015, 12:58 John Karahalis <notifications@github.com> wrote:

> Nice catch. At the moment, *README.md* mainly focuses on using Discord
> and *TESTING.md* mainly focuses on running unit and code quality tests.
> Would *CONTRIBUTING.md* be a good place for this? We have a note here
> <https://github.com/mdn/discord/commit/9c98eb2115d61f0222a12a5e36cbab54bc3fa4d4#diff-6a3371457528722a734f3c51d9238c13R14>
> but could always expand that explanation.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mdn/discord/issues/150#issuecomment-131195357>.
>
-- 

-------------------------------------------
Q: Why is this email five sentences or less?
A: http://five.sentenc.es

**Comment by openjck** [comment]
Ah, good point. A credit card number is required when installing Redis (for verification) but the charges are different. Sounds good to me.

**Comment by openjck** [comment]
I think we neglect to mention scaling up a worker, too. We could add both those notes in one PR.
